iai_donbot_description/robots/iai_donbot.xacro

```diff
@@ -1,279 +1,239 @@
 <?xml version="1.0"?>
```

Comment:
There are a lot of changes here. Can you please describe whats going on?

Reply:
sorry because of my auto-format the tab-space is increase from 2 to 4, it looks like I changed everything but it wasn't. I can change it to 2 if necessary. The code is the same, I just did the following things:
1) Remove gazebo related nodes
2) Make it a macro, so we can call it as <xacro:iai_donbot odom="True"/>
I still keep the old comments, I think we will need to remove them if possible

Reply:
4 would be fine, but if I count correctly, its even 8 at some places. I'd prefer if you'd change it back to 2. Same for iai boxy base 2.

Reply:
I changed it back to 2